pages/_app.tsx

```
@@ -45,6 +46,8 @@ export default class CustomApp extends App<I18nProps> {
 
       if (tips) alert(tips);
     });
+
+    sessionStore.getProfile().catch(console.debug);
```

ESLint

[error] Unexpected console statement. Only these console methods are allowed: warn, error, info.

(no-console)